### ISSUE TITLE
How to add WHERE clause to schema?

### ISSUE THREAD

**Issue by gunererd** [toluaina/pgsync#251]
Sorry if asked before but I couldn't find in examples, documentation or closed issues but how we can add where clause to schema? Say that I'd like to sync only the rows with column _status_ is equal to _100_.

**Comment by toluaina** [comment]
This has come up before. 
Its not easy to come up with a generic user friendly configuration 
based query language to support this functionality.

Your use case is probably straightforward but to completely support 
the `WHERE` clause is not.

The easiest way is to leverage the plugin functionality.
[Here](https://github.com/toluaina/pgsync/blob/master/plugins/sample.py#L24-L42) is an example and [documentation](https://pgsync.com/tutorial/plugins/)

**Comment by toluaina** [comment]
You are right this wasn't so easy.  I have made a  change to how the plugin works in the master branch. So you can do this 
now. Here is an example
```
class MyFilterPlugin(plugin.Plugin):
    """Example Villain plugin."""

    name: str = "Filter"

    def transform(self, doc: dict, **kwargs) -> dict:
        """Demonstrates how to modify a document."""
        doc_id = kwargs["_id"]
        doc_index = kwargs["_index"]
        if doc["id"] == 1:
            return None

        return doc
```
